tests/test_ext_intersphinx.py

```diff
@@ -12,13 +12,18 @@
 import unittest
 
 from docutils import nodes
+from textwrap import dedent
 import mock
+import pytest
+import requests
+from io import BytesIO
```

Comment:
I tried using `six.BytesIO`, and it fails on Python 2. `six.BytesIO` is `StringIO.StringIO` on Python 2, which [doesn't support the `with` statement](http://stackoverflow.com/questions/12028637/pythons-stringio-doesnt-do-well-with-with-statements#12028682), which causes the test to fail.